### PR TITLE
PIMS-24: add the LTSA wms layer using the new url

### PR DESCRIPTION
**Pull request by LawrenceLau2020**
adding new LTSA map layer URL and readme doc for local keycloak configuration.

Please check that the README didn't include anything that's supposed to be confidential.

### CODE REVIEW

File: frontend/src/components/maps/leaflet/LayerPopup/constants/index.tsx
Comment (Fosol) [inline frontend/src/components/maps/leaflet/LayerPopup/constants/index.tsx:14]: We should probably keep this as a separate layer instead of removing it.

**Review by Fosol — APPROVED** [review]
Awesome work!  Just remember to not merge until we have a new signed off `master` branch.

**Review by Fosol — APPROVED** [review]
My only comment was that it may make more sense to still keep the prior layer and add this new one as the default one.  But I'm not sure if the prior layer has any value.

**Comment by LawrenceLau2020** [comment]
> Does it have anything that the new layer doesn't? Are we worried about users who don't have the correct permissions not being able to see the new layer?

I can bring this up with Lindsay and Amit, and get their feedback.

**Comment by LawrenceLau2020** [comment]
> > Does it have anything that the new layer doesn't? Are we worried about users who don't have the correct permissions not being able to see the new layer?
> 
> I can bring this up with Lindsay and Amit, and get their feedback.

It looks like we'll need to keep both layers for now for people who can't view the secured layer, and we will need to figure out a way to give the user the "correct" layer somehow. I'm looking at the axios code and I'm not sure if there's an easy way to do this as the first axios requests makes a "promise" to get the data and returns an error if it doesn't.

**Pull request merged**
